docs/src/main/tut/typeclasses/imports.md

```diff
@@ -40,4 +40,29 @@ val o: Option[String] = None
 o.orEmpty
 ```
 
-**Note**: if you import `cats.implicits._` (the preferred method), you should _not_ also use imports like `cats.syntax.option._` or `cats.instances.either._`. This can result in ambiguous implicit values that cause bewildering compile errors.
+If you'd like to import à la carte, you can do so, by importing from `cats.instances` for the type class instances and `cats.syntax` for syntax enrichment.
+For example, if you'd like to import the `Semigroup` instance for `String` and the corresponding syntax:
+```tut:book
+import cats.instances.string._
+import cats.syntax.semigroup._
+
+"Hello, " |+| "World!"
+```
+The first import pulls the `Semigroup` instance for String into the scope, while the second import adds the `|+|` syntax.
+
+You can also import all syntax or all instances by importing `cats.instances.all._` or `cats.syntax.all._` respectively.
```

Comment:
Isn't this mixed up? I.e., it should be something like "(syntax or instances) ... (`cats.syntax.all._` or `cats.instances.all._`) respectively". (Although english is not my first language.)